esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -3179,11 +3190,16 @@ void getWorkunitCluster(IEspContext &context, const char *wuid, SCMStringBuffer
     }
 }
 
-void getFileResults(IEspContext &context, const char *logicalName, const char *cluster, __int64 start, unsigned &count, __int64 &total,
+void CWsWorkunitsEx::getFileResults(IEspContext &context, const char *logicalName, const char *cluster, __int64 start, unsigned &count, __int64 &total,
     IStringVal &resname, bool bin, IArrayOf<IConstNamedValue> *filterBy, MemoryBuffer &buf, bool xsd)
 {
     Owned<IResultSetFactory> resultSetFactory = getSecResultSetFactory(context.querySecManager(), context.queryUser(), context.queryUserId(), context.queryPassword());
-    Owned<INewResultSet> result(resultSetFactory->createNewFileResultSet(logicalName, cluster));
+    Owned<IDistributedFile> df = resultSetFactory->lookupLogicalName(logicalName);
```

Comment:
see other comments, IResultSetFactory shouldn't have a lookupLogicalName method (should stay as internal helper), so this code should lookup the logical file as before here.